### ISSUE TITLE
Unable to link with react-native.

### ISSUE THREAD

**Issue by pgarciacamou** [dadevelopers/react-native-opencv#2]
I tried multiple different ways to link `pod` branch into my react-native project:

1. using `react-native link`
2. manually installing `react-native-opencv` following react-native tutorials.
3. manually installing following the readme file in the branch.
4. Rebuilding this module.

Nothing worked. It always ended giving me the following message:

<img width="370" alt="screen shot 2017-02-15 at 3 35 51 pm" src="https://cloud.githubusercontent.com/assets/8354571/23000259/a00e1652-f394-11e6-9f1f-a1541da9e8d1.png">

Basically it is saying that [RNOpenCV](https://github.com/dadevelopers/react-native-opencv/blob/pod/index.js#L8) is undefined. Which I managed to console.log and it turns out it is definitely undefined.

I didn't find anywhere how to solve this issue :/
